### ISSUE TITLE
Add a way to specify which servers receive the network join message OR add an option to send it to only the global chat channel

### ISSUE THREAD

**Issue by Pemigrade** [MultiChat/Development#134]
**Is your feature request related to a problem? Please describe.**
I have four servers on my network: a survival server, a creative server, a minigames server, and a hub server. The hub server, creative server, and the minigames server are all in the forced global chat channel. Whenever someone joins, people always say "welcome back." However, users that say "welcome back" don't get their welcome back seen by the user that joined the network if they're on the minigames server. It's confusing to explain to users why they need to ignore the join messages on the minigame server but should say welcome back on the other servers, and there's really no way to disable the messages on the minigame server either.

**Describe the solution you'd like**
A way to specify which servers receive the network join message

**Describe alternatives you've considered**
There are some message filter plugins, but those don't seem to be updated and they're impossible to use since my network join format is identical to a separate join message I have configured for the minigames server.

**Additional context**
Whenever I say join messages in this issue, I mean both join and quit messages.

**Comment by OllieMartin** [comment]
I agree that network join and leave messages should only be shown in the global chat servers. I will make sure this is changed in v1.10 so any servers that are excluded from global chat do not receive the messages.

**Comment by OllieMartin** [comment]
Hey there,

I went to implement this and was unsure what the intended functionality would be so I thought I'd ask your opinion as the original asker.

If a player joins on a server which is NOT in the global chat, then the join message would not be displayed. However, how would this then work when they moved to a server that is part of the global chat? I feel that it would be confusing to say "PLAYER has joined the network" when they were already on the network.

What was your original intention? Would you just have no message shown for them?

Thanks, I appreciate your thoughts on this.

**Comment by Pemigrade** [comment]
Sorry! I had typed up a response for this but it looks like I forgot to send it.

On my server, all players are forced to join the hub server. Therefore, all network join messages will be for a player joining the hub server from the Minecraft launcher. So, I wouldn't have the issue where a player joins a server which is NOT in the global chat.

For networks which have servers that are not in the global chat, I think it'd make the most sense to not display the network join message at all. The network join message should only be displayed to the global chat if the user joins a server that places him in the global chat channel.

For example, if a network has a server A, server B, and server C, and players can ONLY talk in global chat in server A and server B, it should only display the message if the player logs on in server A or server B. If they join server C, there should be no message.

It gets a little more confusing when players are able to choose which channel they speak in. If they get to choose, the global join message should only be broadcast if the user joins while having the global chat channel selected.

**Comment by OllieMartin** [comment]
Additional use case in: #140
